src/reflect/scala/reflect/io/ZipArchive.scala

```diff
@@ -62,6 +62,25 @@ object ZipArchive {
       if (front) path.substring(0, idx + 1)
       else path.substring(idx + 1)
   }
+  def pathToDotted(path: String): String = {
+    if (path == "/") ""
```

Comment:
Can we assume / as path separator here?

Reply:
Yep, it comes from the line `val root = new DirEntry("/")` elsewhere in this file.

Reply:
@retronym Good. Perhaps add a constant for it so it isn't correct by correlation? :)